### ISSUE TITLE
Question:  High Availability

### ISSUE THREAD

**Issue by joestafford** [google/metallb#224]
**Is this a bug report or a feature request?**:
Let's call this a question!

**What happened**:
How is high availability addressed in this load balancing service?

**What you expected to happen**:
If a k8s node fails and is the elected leader with IPs, another node quickly takes leadership.

**How to reproduce it (as minimally and precisely as possible)**:
Have not yet attempted to test this, again a question which I've yet to find covered in documentation.

**Anything else we need to know?**:
Love this project!

**Environment**:
- MetalLB version: 0.5.0
- Kubernetes version: 1.10
- BGP router type/version:  ARP configuration
- OS (e.g. from /etc/os-release):  Ubuntu 16.04
- Kernel (e.g. `uname -a`):  4.4.0-116-generic #140-Ubuntu SMP

**Comment by joestafford** [comment]
Just checking in.

**Comment by danderson** [comment]
Apologies for the delay! I started writing a reply, and then brilliantly forgot to send it.

TL;DR: yes, MetalLB handles HA.

In BGP mode, all nodes run BGP sessions, so there is naturally redundancy (and true load-balancing) across nodes.

In layer2 mode, one node is elected leader, and "owns" all the service IPs for the cluster. If that node fails, leadership election times out after 10s, at which point another node gets elected and takes over the announcements.

For layer2, #195 may in future further reduce the impact of node failure, by distributing ownership of IPs across the cluster more - so that a node failure is only a temporary outage for some IPs, rather than all.

You make a valid point that this behavior is not documented anywhere on the website. I'll convert this issue into a documentation bug to fix that :).
